src/utils.test.tsx

```diff
@@ -364,3 +365,23 @@ describe('checkForDuplicates', () => {
     expect(result).toEqual(['3', '1', '4', '2']);
   });
 });
+
+describe('getNonEmptyString', () => {
+  it('should return the string for non-empty strings', () => {
+    expect(getNonEmptyString('Hello')).toBe('Hello');
+    expect(getNonEmptyString('   Hello   ')).toBe('   Hello   ');
```

Comment:
Should this not also trim the spaces off the string like the other dialogues do? Function would probably need trimmed in the name if it did.